packages/remirror__extension-entity-reference/src/entity-reference-extension.ts

```diff
@@ -169,6 +169,34 @@ export class EntityReferenceExtension extends MarkExtension<EntityReferenceOptio
     };
   }
 
+  /**
+   * Dispatch a transaction that selects the range of the entity reference then scrolls to it.
+   *
+   * @param entityReferenceId - The entity's reference Id.
+   *
+   * @returns True if the scrolling was applied, else it returns false
+   *
+   */
+  @command()
+  scrollToEntityReference(entityReferenceId: string): CommandFunction {
```

Comment:
I'm not sure how to unit test this command. Is there an example of testing if selection/scrolling worked?

Reply:
For scrolling, I don't know how to test it. It usually won't go wong so it's ok to skip it. 

For selection, you can search the keyword "selection" in the following files:

- packages/remirror__extension-list/__tests__/list-commands.spec.ts
- packages/remirror__extension-link/__tests__/link-extension.spec.ts